torch/csrc/jit/interpreter.cpp

```diff
@@ -949,7 +949,9 @@ struct CodeImpl {
     };
     auto & inst = instructions.at(pc);
     writeList(inst.outputs);
-    out << " = " << inst.debug_name.toString() << " ";
+    // TODO: Maybe this wants to be unqualified.  Not sure what is going on
+    // with debug_name
+    out << " = " << inst.debug_name.toQualString() << " ";
```

Comment:
`debug_name` is just the kind of the operator that was used to select the dispatch. We usually print that unqualified (e.g. when we dump the graph)